PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -0,0 +1,98 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+from PhysicsTools.NanoAOD.jets_cff import *
+from PhysicsTools.NanoAOD.muons_cff import *
+from PhysicsTools.NanoAOD.taus_cff import *
+from PhysicsTools.NanoAOD.electrons_cff import *
+from PhysicsTools.NanoAOD.photons_cff import *
+from PhysicsTools.NanoAOD.globals_cff import *
+from PhysicsTools.NanoAOD.genparticles_cff import *
+from PhysicsTools.NanoAOD.vertices_cff import *
+from PhysicsTools.NanoAOD.met_cff import *
+from PhysicsTools.NanoAOD.triggerObjects_cff import *
+from PhysicsTools.NanoAOD.isotracks_cff import *
+from PhysicsTools.NanoAOD.NanoAODEDMEventContent_cff import *
+
+nanoMetadata = cms.EDProducer("UniqueStringProducer",
+    strings = cms.PSet(
+        tag = cms.string("untagged"),
+    )
+)
+
+linkedObjects = cms.EDProducer("PATObjectCrossLinker",
+   jets=cms.InputTag("finalJets"),
+   muons=cms.InputTag("finalMuons"),
+   electrons=cms.InputTag("finalElectrons"),
+   taus=cms.InputTag("finalTaus"),
+   photons=cms.InputTag("finalPhotons"),
+)
+
+simpleCleanerTable = cms.EDProducer("NanoAODSimpleCrossCleaner",
+   name=cms.string("cleanmask"),
+   doc=cms.string("simple cleaning mask with priority to leptons"),
+   jets=cms.InputTag("linkedObjects","jets"),
+   muons=cms.InputTag("linkedObjects","muons"),
+   electrons=cms.InputTag("linkedObjects","electrons"),
+   taus=cms.InputTag("linkedObjects","taus"),
+   photons=cms.InputTag("linkedObjects","photons"),
+   jetSel=cms.string("pt>15"),
+   muonSel=cms.string("isPFMuon && innerTrack.validFraction >= 0.49 && ( isGlobalMuon && globalTrack.normalizedChi2 < 3 && combinedQuality.chi2LocalPosition < 12 && combinedQuality.trkKink < 20 && segmentCompatibility >= 0.303 || segmentCompatibility >= 0.451 )"),
+   electronSel=cms.string(""),
+   tauSel=cms.string(""),
+   photonSel=cms.string(""),
+   jetName=cms.string("Jet"),muonName=cms.string("Muon"),electronName=cms.string("Electron"),
+   tauName=cms.string("Tau"),photonName=cms.string("Photon")
+)
+
+
+genWeightsTable = cms.EDProducer("GenWeightsTableProducer",
+    genEvent = cms.InputTag("generator"),
+    lheInfo = cms.InputTag("externalLHEProducer"),
+    preferredPDFs = cms.vuint32(91400,260001),
+    namedWeightIDs = cms.vstring(),
+    namedWeightLabels = cms.vstring(),
+    lheWeightPrecision = cms.int32(14),
+    maxPdfWeights = cms.uint32(50), # for NNPDF, keep only the first 50 replicas (save space)
+    debug = cms.untracked.bool(False),
+)
+lheInfoTable = cms.EDProducer("LHETablesProducer",
+    lheInfo = cms.InputTag("externalLHEProducer"),
+)
+
+l1bits=cms.EDProducer("L1TriggerResultsConverter", src=cms.InputTag("gtStage2Digis"), legacyL1=cms.bool(False))
+
+nanoSequence = cms.Sequence(
```

Comment:
use Task here?

Reply:
unless filters are explicitly involved, here and in other places, better switch to using tasks

Reply:
can i then turn a task into a path?

Reply:
yes 
```
aTask = cms.Task(modA, modB, ... )
aSeq = cms.Sequence(aTask)
aPath = cms.Path(aSeq)
```